services/server/src/handlers/user.ts

```diff
@@ -424,7 +419,7 @@ async function findOnboardedTutors(req: Request, res: Response) {
     : tutors;
   const ordered = orderTutors({
     tutors: filtered,
-    rules,
+    slots: [], // TODO: retrieve AvailabilitySlots from db
```

Comment:
We can avoid ordering tutors by their rules/slots for now. 

Update the `orderTutors` function to not to use `rules/slots`.